zhaquirks/schneider/shutter.py

```diff
@@ -0,0 +1,125 @@
+"""Module to handle quirks of the Schneider Electric shutter.
+
+manufacturer specific attributes to control displaying and specific configuration.
+"""
+
+from zigpy.quirks import CustomCluster, CustomDevice
+import zigpy.types as t
+from zigpy.zcl.clusters.closures import WindowCovering
+from zigpy.zcl.clusters.general import (
+    Basic,
+    Groups,
+    Identify,
+    LevelControl,
+    OnOff,
+    Ota,
+    Scenes,
+)
+from zigpy.zcl.clusters.homeautomation import Diagnostic
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+
+
+class SchneiderShutterCluster(CustomCluster, WindowCovering):
+    """Schneider custom cluster."""
+
+    manufacturer_attributes = {
+        0xE014: ("LiftDriveUpTime", t.uint16_t),
+        0xE015: ("LiftDriveDownTime", t.uint16_t),
+        0xE016: ("TiltOpenCloseAndStepTime", t.uint16_t),
+        0xE017: ("TiltPositionPercentageAfterMoveToLevel", t.uint8_t),
+    }
+
+
+class SchneiderShutter(CustomDevice):
+    """SchneiderShutter custom device."""
+
```

Comment:
can you include the simple descriptor output comments please

Reply:
@zoic21 I want to cut a release before the beta on Wednesday. Any chance you can address this?

Reply:
Hello,
Unfortunately I had to return the loan equipment to the manufacturer which means that I no longer have access to its descriptor....

Reply:
It's done